### PR TITLE
Tmsv patch 2

### PR DESCRIPTION
**Pull request by tmsv**

### CODE REVIEW

**Comment by tmsv** [comment]
@sobolevn Hi, I made it in second easy way. I will also deal with the first correct way when I have the opportunity.
Please, tell me if something needs to be corrected or remade. I appreciate the tips that you give.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+flake8-blind-except = "^0.1.1"
+flake8-docstrings = "^1.3"
+flake8-string-format = "^0.2.3"
+flake8-coding = "^1.3"
+flake8-module-name = "^0.1.5"
+flake8-pytest = "^1.3"
+pep8-naming = "^0.7.0"
+pytest = "^3.8"
+pytest-flake8 = "^1.0"
+requests = "^2.19"
```
Comment (sobolevn) [inline pyproject.toml]: We do not need this dependency.

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+pendulum = "^2.0"
```
Comment (sobolevn) [inline pyproject.toml]: We do not need this dependency.

**Comment by tmsv** [comment]
Thank you for guiding. I'll close this pull request and I'll create new one to save time.

**Comment by sobolevn** [comment]
@tmsv ok, but next time I will ask you to update the existing one.

Because currently you make a mistake by creating a new one:
1. we loose context, why the changes were made, who asked for them, etc
2. we pollute the repository with the same code changes, they will be tracked in search

**Pull request closed without merging**